Floofbot/Services/Repository/Models/ServerConfig.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Floofbot.Services.Repository.Models
+{
+    public partial class AdminConfig
+    {
+        [Key]
+        public ulong ServerId { get; set; }
+        public ulong MuteRoleId { get; set; }
+
```

Comment:
Remove blank line here. Also remove the extraneous includes (lines 1 and 2, probably).